### ISSUE TITLE
How to transmit parameters?

### ISSUE THREAD

**Issue by 6firefly** [vitalets/x-editable-yii#65]
If I have something like:

array(
            'class' => 'editable.EditableColumn',
            'name'=>'no_search',
            'value'=>'isset($data->idAuto)?$data->idAuto->no:""',
            'editable' => array(    //editable section
                  'type'       => 'text',
                  'attribute'  => 'idAuto.no',
                  'url'        => $this->createUrl('auto/updateNo'),
                  'placement'  => 'right',
              ) ,
        ),

in a TbGridView, how do I transmit $data->id as parameter?

**Comment by vitalets** [comment]
Do you mean you need the id of parent record when updating child `idAuto`?
Could you explain a bit why? 
As you have `pk = idAuto.primaryKey` that should be enough..

**Comment by 6firefly** [comment]
I know about pk = idAuto.primaryKey, but I also need the parent id ( $data->id).  If the $data->idAuto->no is changed, then I need to see if the new value need to have or not a new parent (simple put).

**Comment by vitalets** [comment]
ok. it was a missing feature. I've fixed in dev branch,
https://github.com/vitalets/x-editable-yii/tree/dev
you can use it as it is only change from released 1.3.1.
so, it should be:

``` php
array(
'class' => 'editable.EditableColumn',
'name'=>'no_search',
'value'=>'isset($data->idAuto)?$data->idAuto->no:""',
'editable' => array( 
  'type' => 'text',
  'attribute' => 'idAuto.no',
  'url' => $this->createUrl('auto/updateNo'),
  'placement' => 'right',
  'params' => array('id' => '$data->id'), //will be evaluated
) ,
),
```

**Comment by 6firefly** [comment]
The expression is evaluated the same for every field/row; so I have the same id for row 1 as for row 10 for example... the values should be different... 

so if I'll have:

array(
'class' => 'editable.EditableColumn',
'name'=>'no_search',
'value'=>'$data->id',
'editable' => array( 
  'type' => 'text',
  'attribute' => 'idAuto.no',
  'url' => $this->createUrl('auto/updateNo'),
  'placement' => 'right',
  'params' => array('id' => '$data->id'), //will be evaluated
) ,
),

the value is different form id param; this should not happen...
